client/src/components/Dashboard.tsx

```diff
@@ -139,13 +141,27 @@ const Dashboard = () => {
     }
   }
 
-  function renderSchedule(schedule: Schedule, viewMode: string) {
+  function sessionIsOver(end: string): boolean {
```

Comment:
Since the function is called `sessionIsOver`, I would expect a session as input here.
```suggestion
  function sessionIsOver({ end }: Session): boolean {
```